packages/tofu/src/inspectSource.js

```diff
@@ -183,37 +183,110 @@ function inspectGlobals(
  * @param {import('@babel/types').Node} ast
  * @param {string[]} [packagesToInspect] - List of module IDs to look for; if
  *   omitted, will inspect all imports
+ * @param {boolean} [deep] Return named exports and imports
  * @returns {string[]} - List of imported identifiers
  */
-function inspectEsmImports(ast, packagesToInspect) {
+function inspectEsmImports(ast, packagesToInspect, deep = true) {
```

Comment:
appreciate the opt-in for backwards cmopatibility, but I think we should turn it on in lavamoat-node classic too if circumstances permit.